CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
 ## 14.1.4
 
 ### Features
+- `[lf-login]`: Add function `beforeFetchRequestAsync` to login component
+- `[lf-login]`: Add function `afterFetchResponseAsync` to login component
+- `[lf-login]`: Add property `authorizationRequestHandler` to login component
+- `[lf-login-types]` Added new types to support `authorizeRequestHandler`
```

Comment:
only need to add the public facing interface
you could just do 
- `[lf-login]`: Add property `authorizationRequestHandler` to lf-login component and added related types `LfHttpRequestHandler` and `LfBeforeFetchResult`